chemreg/substance/tests/test_serializers.py

```diff
@@ -55,3 +56,15 @@ def test_substance_type(substance_type_factory):
     serializer = substance_type_factory.build()
     assert serializer.is_valid()
     serializer.save()
+
+
+@pytest.mark.django_db
+def test_synonym_quality_serializer():
+    assert issubclass(SynonymQualitySerializer, HyperlinkedModelSerializer)
+
```

Comment:
Could you add an additional test for the validation?

```suggestion


@pytest.mark.django_db
def test_synonym_quality_score_weight_validation(synonym_quality_factory):
    model_dict = synonym_quality_factory.build().initial_data
    # Negative score_weights are forbidden
    model_dict.update(score_weight=-1.0)
    assert not SynonymQualitySerializer(data=model_dict).is_valid()

```